examples/serverless/main.tf

```diff
@@ -19,8 +19,9 @@ data "aws_subnet_ids" "all" {
 module "aurora" {
   source                = "../../"
   name                  = "aurora-serverless"
-  engine                = "aurora"
+  engine                = "aurora-postgresql"
   engine_mode           = "serverless"
+  engine_version        = "10.7"
```

Comment:
I'm afraid this parameter is not applicable to aurora serverless. It is incremented by AWS without our control. Sounds like we should rather ignore it when creating the cluster. 

Thoughts? @a7i

Reply:
My experience is exactly the opposite. `engine_version` does work with serverless and in fact, for `aurora-postgresql` it is required. 

```
module "rds-aurora" {
  source  = "terraform-aws-modules/rds-aurora/aws"
  version = "~> 3.0.0"
  ...
  engine                = "aurora-postgresql"
  engine_mode           = "serverless"
  engine_version        = "10.12"
  ...
}

resource "aws_db_parameter_group" "serverless_db_parameter_group" {
  name   = "serverless-db-parameter-group"
  family = "aurora-postgresql10"
  tags   = local.tags
}

resource "aws_rds_cluster_parameter_group" "serverless_cluster_parameter_group" {
  name   = "serverless-cluster-parameter-group"
  family = "aurora-postgresql10"
  tags   = local.tags
}
```

You can use `aws rds --region eu-central-1 describe-db-engine-versions` to get more information about the supported `Engine`, `EngineVersion`, `DBParameterGroupFamily` and `SupportedEngineModes`.

Reply:
@ppihus But you're specifying `10.12`. What happens when AWS updates it from `10.12` to `10.13`?
I'm saying that we should set that value dynamically when creating, but then we should ignore it during all other lifecycles.

I hope it makes sense.

Reply:
Are you saying that AWS supports only one engine version at a time for serverless-postgresql?

So when we use dynamic value and then AWS upgrades to `10.12-->10.13`, but for some specific reason I still need to create a cluster with `10.12`, I wouldn't be able to do that?

I checked `us-east-1` available db engine versions as well. For MySQL there seems to be multiple versions available, but for postgres there is currently only one indeed 🤔

Reply:
It was over a year ago, but I remember that lowest version was auto-upgraded, or something like that. It may not be valid, needs some testing.